docs/technical-setup/environment-setup/install-r-rstudio.md

```diff
@@ -42,26 +42,106 @@ These tools are available from the [R for macOS website](https://mac.r-project.o
 1. Install the **GNU Fortran compiler**.
 Click the link on the [Tools - R for Mac OS](https://mac.r-project.org/tools/) page to download the `gfortran` installer package and follow all installation instructions.
 
-<!--
-### Install R on Windows
+### Install R on Windows with WSL2
 
+These instructions will install R on the WSL2 side of your computer.
 
-1. Navigate to the [Windows download page on the CRAN website](https://cran.r-project.org/bin/windows/base/), and follow instructions to download and install R.
+1. Copy and paste this code into the Ubuntu terminal to install R:
+
+    ```sh
+    # install a few helper packages needed to install R and certain R packages
+    sudo apt install --no-install-recommends software-properties-common dirmngr libssl-dev
+
+    # add the signing key (by Michael Rutter) for these repos
+    wget -qO- https://cloud.r-project.org/bin/linux/ubuntu/marutter_pubkey.asc | sudo tee -a /etc/apt/trusted.gpg.d/cran_ubuntu_key.asc
+
+    # add the R 4.0 repo from CRAN -- adjust 'focal' to 'groovy' or 'bionic' as needed
+    sudo add-apt-repository "deb https://cloud.r-project.org/bin/linux/ubuntu jammy-cran40/"
+
+    # install R
+    sudo apt install --no-install-recommends r-base r-base-dev
+    ```
+
+1. To confirm R was successfully installed, run `R` in the Ubuntu terminal.
+This should launch the R console in terminal; hit `Ctrl+D` to quit.
+    - If you instead get an error message that the command was not found, your R installation was not successful; please feel free to use the [Troubleshooting GitHub Discussions](../../communications-tools/index.md#ask-questions) to get some help!
+
+1. Finally, copy and paste this line into the Ubuntu terminal.
+    ```sh
+    echo "\noptions(repos = list(CRAN='https://p3m.dev/cran/__linux__/jammy/latest'))" >> ~/.Rprofile
```

Comment:
Echo in my testing on ubuntu leaves the literal `\n`. I don't think it should be needed in most cases, especially with new installs like this.
```suggestion
    echo "options(repos = list(CRAN='https://p3m.dev/cran/__linux__/jammy/latest'))" >> ~/.Rprofile
```

If you really want it, `echo -e` will be safer, or `echo ""; echo "next line"`

Finally, you could make this version agnostic too:

```
 echo "options(repos = list(CRAN='https://p3m.dev/cran/__linux__/$(lsb_release -cs)/latest'))" >> ~/.Rprofile
```